doc/en/api/1.0.0/urlchecks.yaml

```diff
@@ -0,0 +1,239 @@
+---
```

Comment:
This definition fails validation when pasted in the [swagger editor](https://editor.swagger.io/):

![image](https://github.com/geoserver/geoserver/assets/325433/89dbd488-b96e-4a13-b8cb-2bfb92aef71b)

Reply:
Fixed. Thanks.